components/Starknet/modules/architecture-and-concepts/pages/smart-contracts/compiled_class_hash.adoc

```diff
@@ -0,0 +1,39 @@
+[id="compiled_class_hash"]
+= Compiled class hash
+
+
+The compiled class hash is a cryptographic hash that results from the compilation process of a Cairo class
```

Comment:
@raizo07  Can you start the topic by stating why the reader should care about it? Is it something that they will use in some way? If not, then who does use it and for what purpose?